physics/ksp_system_test.cpp

```diff
@@ -0,0 +1,469 @@
+    minmus.elements.mean_motion = +5.83228807719951003e-06 * (Radian / Second);
+    minmus.elements.inclination = +1.04719755119659780e-01 * Radian;
+    minmus.elements.longitude_of_ascending_node =
+        +1.36135681655557694e+00 * Radian;
+    minmus.elements.argument_of_periapsis = +6.63225115757845263e-01 * Radian;
+    minmus.elements.mean_anomaly = +8.99999976158141979e-01 * Radian;
+    mun.elements.eccentricity = +0.00000000000000000e+00;
+    mun.elements.mean_motion = +4.52078533000627999e-05 * (Radian / Second);
+    mun.elements.inclination = +0.00000000000000000e+00 * Radian;
+    mun.elements.longitude_of_ascending_node =
+        +0.00000000000000000e+00 * Radian;
+    mun.elements.argument_of_periapsis = +0.00000000000000000e+00 * Radian;
+    mun.elements.mean_anomaly = +1.70000004768372004e+00 * Radian;
+    eve.elements.eccentricity = +9.99999977648258036e-03;
+    eve.elements.mean_motion = +1.11049676511037010e-06 * (Radian / Second);
+    eve.elements.inclination = +3.66519126274052684e-02 * Radian;
+    eve.elements.longitude_of_ascending_node =
+        +2.61799387799149408e-01 * Radian;
+    eve.elements.argument_of_periapsis = +0.00000000000000000e+00 * Radian;
+    eve.elements.mean_anomaly = +3.14000010490416992e+00 * Radian;
+    gilly.elements.eccentricity = +5.50000011920928955e-01;
+    gilly.elements.mean_motion = +1.61692985452753988e-05 * (Radian / Second);
+    gilly.elements.inclination = +2.09439510239319560e-01 * Radian;
+    gilly.elements.longitude_of_ascending_node =
+        +1.39626340159546358e+00 * Radian;
+    gilly.elements.argument_of_periapsis = +1.74532925199432948e-01 * Radian;
+    gilly.elements.mean_anomaly = +8.99999976158141979e-01 * Radian;
+    moho.elements.eccentricity = +2.00000002980231989e-01;
+    moho.elements.mean_motion = +2.83568694188237007e-06 * (Radian / Second);
+    moho.elements.inclination = +1.22173047639603072e-01 * Radian;
+    moho.elements.longitude_of_ascending_node =
+        +1.22173047639603061e+00 * Radian;
+    moho.elements.argument_of_periapsis = +2.61799387799149408e-01 * Radian;
+    moho.elements.mean_anomaly = +3.14000010490416992e+00 * Radian;
+  }
+
+  not_null<std::unique_ptr<Ephemeris<KSP>>> MakeEphemeris() {
+    HierarchicalSystem<KSP> hierarchical_system(std::move(sun.owned_body));
+    for (auto const celestial : planets_and_moons) {
+      hierarchical_system.Add(std::move(celestial->owned_body),
+                 celestial->parent->body,
+                 celestial->elements);
+    }
+    HierarchicalSystem<KSP>::BarycentricSystem barycentric_system =
+        hierarchical_system.ConsumeBarycentricSystem();
+    return make_not_null_unique<Ephemeris<KSP>>(
+        std::move(barycentric_system.bodies),
+        std::move(barycentric_system.degrees_of_freedom),
+        ksp_epoch,
+        integrators::McLachlanAtela1992Order5Optimal<Position<KSP>>(),
+        45 * Minute,
+        1 * Milli(Metre));
+  }
+
+  Instant const ksp_epoch;
```

Comment:
Could we put a trailing _ for the member variables?  VS should be able to help with the renaming.